src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs

```diff
@@ -183,7 +184,7 @@ public Task GetClusterTopology()
                 if (topology.Members.Count == 0)
                 {
                     var tag = ServerStore.NodeTag ?? "A";
-                    var serverUrl = ServerStore.NodeHttpServerUrl;
+                    var serverUrl = ServerStore.GetNodeHttpServerUrl(HttpContext.Request.GetClientRequestedNodeUrl());
```

Comment:
@ayende Check for public server url setting is made in one place (within this method). Please see https://github.com/ravendb/ravendb/pull/4239/files#diff-99656d10b23ccec3011c0af6f6cbe346R1568